src/styles/baseThemes/lightBaseTheme.js

```diff
@@ -26,6 +26,8 @@ export default {
     accent1Color: pinkA200,
     accent2Color: grey100,
     accent3Color: grey500,
+    accent4Color: cyan500,
```

Comment:
I have two question here.
- Why do we increase the number of accent color? 
- Isn't it going to be a breaking changes as the value is not longer derivated from the primary color?